### PR TITLE
feature: frisby.options()

### PR DESCRIPTION
**Pull request by koooge**
Hi there. I would like to test OPTIONS using frisby. (And I will create PR frisby.head() if this PR is merged)

### CODE REVIEW

File: src/frisby/spec.js [hunk 1]
```diff
@@ -160,6 +160,13 @@ class FrisbySpec {
     return this;
   }
 
+  /**
+   * OPTIONS convenience wrapper
+   */
+  options(url, params) {
+    return this._requestWithBody('OPTIONS', url, params);
```
Comment (H1Gdev) [inline src/frisby/spec.js]: HTTP OPTIONS method has no body.
https://developer.mozilla.org/docs/Web/HTTP/Methods/OPTIONS

Comment (koooge) [reply src/frisby/spec.js]: fixed :+1:

**Comment by vlucas** [comment]
Looks good to me. Thanks for the contribution! 👍

**Comment by roydekleijn** [comment]
When will this be released

**Pull request merged**